ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -106,12 +95,16 @@
         concept-sub-type (get-sub-concept-type-concept-id-prefix spec-key spec-version)]
     (if (not= concept-type spec-key)
       (throw UnsupportedOperationException)
-      {:concept (assoc {} :metadata raw-document
+      {:concept (assoc {}
+                       :metadata raw-document
                        :provider-id provider-id
+                       :concept-type concept-type
                        :format (str "application/vnd.nasa.cmr.umm+json;version=" spec-version)
                        :native-id native-id
                        :user-id (api-core/get-user-id request-context headers)
-                       :extra-fields {}
+                       :extra-fields {:document-name (:Name document)
+                                      :schema spec-key
+                                      :mime-type (str "application/vnd.nasa.cmr.umm+json;version=" spec-version)}
```

Comment:
why is this the correct mime type and not something constructed? Is this what is returned to the user in a header?

Reply:
The return type depends on what the user asked for. The mime-type needs to be what has been already defined, unless we create a new one.  A lot of places in the code depend on the specific mime-type.  If you want to change this I would rather you create a ticket to change it.  It will be 3-5 days worth of work.

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8614